workflow/util/util.go

```diff
@@ -699,7 +699,7 @@ func FormulateResubmitWorkflow(ctx context.Context, wf *wfv1.Workflow, memoized
 	onExitNodeName := wf.ObjectMeta.Name + ".onExit"
 	err := packer.DecompressWorkflow(wf)
 	if err != nil {
-		log.Fatal(err)
+		log.Error(err)
```

Comment:
these aren't equivalent -- `log.Error` doesn't `exit`. so we should either return the error or panic. for equivalency with the previous logic, it would be `log.Panic`

Reply:
OK, changed